filesystem/iso9660.ksy

```diff
@@ -354,6 +354,102 @@ types:
         type: u1
       - id: offset
         type: s1
+  rrip_nm:
+    doc-ref: rrip 4.1.4
+    seq:
+      - id: length
+        type: u1
+      - id: version
+        contents: [ 0x1 ]
+      - id: flags # ToDo, made it too slow for now
+        type: u1
+      - id: name
+        type: str
+        encoding: ASCII # Lookup POSIX:2.2.2.60
+        size: length - 5
+  rrip_px:
+    doc-ref: rrip 4.1.1
+    seq:
+      - id: length
+        type: u1
+      - id: version
+        contents: [ 0x1 ]
+      - id: file_mode
+        type: u4bi
+      - id: links
+        type: u4bi
+      - id: user
+        type: u4bi
+      - id: group
+        type: u4bi
+      - id: serial
+        type: u4bi
+        if: length >= 44
+  rrip_tf:
+    doc-ref: rrip 4.6.1
+    seq:
+      - id: length
+        type: u1
+      - id: version
+        contents: [ 0x1 ]
+      - id: long_form
+        type: b1
+      - id: effective
+        type: b1
+      - id: expiration
+        type: b1
+      - id: backup
+        type: b1
+      - id: attributes
+        type: b1
+      - id: access
+        type: b1
+      - id: modify
+        type: b1
+      - id: creation
+        type: b1
+      - id: creation_short
+        type: datetime_short
+        if: ( creation == true ) and ( long_form == false )
```

Comment:
I see lots of ```long_form == true``` and ```long_form == false```.
Consider moving long and short forms into a separate type.